Failing to build ruby 2.0.0-p247 on Mavericks

Here's the full error trace:

https://gist.github.com/jamonholmgren/7141903

Error portion:

```
generating makefile exts.mk
Failed to configure -test-/win32/dln. It will not be installed.
Failed to configure -test-/win32/fd_setsize. It will not be installed.
Failed to configure dl/win32. It will not be installed.
Failed to configure fiddle/win32. It will not be installed.
Failed to configure tk. It will not be installed.
Failed to configure tk/tkutil. It will not be installed.
Failed to configure win32ole. It will not be installed.
linking shared-object -test-/array/resize.bundle
ld: can't open output file for writing: ../../../../.ext/x86_64-darwin13.0.0/-test-/array/resize.bundle, errno=2 for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [../../../../.ext/x86_64-darwin13.0.0/-test-/array/resize.bundle] Error 1
make[1]: *** [ext/-test-/array/resize/all] Error 2
make: *** [build-ext] Error 2
!!! Compiling ruby 2.0.0-p247 failed!
```

I was able to install 2.0.0 using the chruby setup.sh method, but then Rails kept throwing a ruby error, so I deleted it and am trying again using ruby-install and getting this error.

Ideas?

Jamon

--- a/share/ruby-install/mruby/functions.sh
+++ b/share/ruby-install/mruby/functions.sh
@@ -10,6 +10,9 @@ ruby_url="${ruby_url:-$ruby_mirror/$ruby_version/$ruby_archive}"
 #
 function compile_ruby()
 {
+	log "Cleaning mruby $ruby_version ..."
+	make clean || return $?
+
 	log "Compiling mruby $ruby_version ..."
 	make "${make_opts[@]}" || return $?
 }

--- a/share/ruby-install/rbx/functions.sh
+++ b/share/ruby-install/rbx/functions.sh
@@ -43,6 +43,9 @@ function configure_ruby()
 				    "${configure_opts[@]}" || return $?
 			;;
 	esac
+
+	log "Cleaning rubinius $ruby_version ..."
+	bundle exec rake distclean || return $?
 }
 
 #

--- a/share/ruby-install/ruby/functions.sh
+++ b/share/ruby-install/ruby/functions.sh
@@ -28,6 +28,9 @@ function configure_ruby()
 				    "${configure_opts[@]}" || return $?
 			;;
 	esac
+
+	log "Cleaning ruby $ruby_version ..."
+	make clean || return $?
 }
 
 #

